src/Commands/WhatsAppWehhook.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace CarroPublic\CarroMessenger\Commands;
+
+use MessageBird\Client;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Log;
+use MessageBird\Objects\Conversation\Webhook;
+
+class WhatsAppWehhook extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'whatsapp:message-bird:webhooks';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'To manage whatsapp messagebird webhooks';
+
+    /**
+     * Toky app url
+     *
+     * @var string
+     */
+    protected $appUrl;
+
+     /**
+     * MessageBird client
+     *
+     * @var \MessageBird\Client
+     */
+    private $messageBirdClient;
+
+     /**
+     * WhatsApp ChannelId
+     *
+     * @var string
+     */
+    private $whatsAppchannelId;
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $this->appUrl = config('carro_messenger.webhook_url.webhook_url');
+
+        $this->messageBirdClient    = new Client(config('carro_messenger.message_bird.access_key'));
+
+        $this->whatsAppchannelId    = config('carro_messenger.message_bird.whatsapp_channel_id');
+
+        $actions = ['list', 'get', 'create', 'delete'];
+
+        $action = $this->choice('What kind of event action you want to perform', $actions, 'list');
+
+        call_user_func(__NAMESPACE__ . '\WhatsAppWehhook::'.$action);
+    }
+
+    /**
+     * Performing create a new toky event
+     */
+    private function create()
+    {
+        $events = [
+            Webhook::EVENT_CONVERSATION_CREATED,
+            Webhook::EVENT_CONVERSATION_UPDATED,
+            Webhook::EVENT_MESSAGE_CREATED,
+            Webhook::EVENT_MESSAGE_UPDATED,
+        ];
+
+        $event = $this->choice('What kind of event you want to create', $event);
+        $webhookUrl = $this->ask('Please enter the webhook URL');
+
+        $webhook = new Webhook();
+        $webhook->events    = $events;
+        $webhook->channelId = $this->whatsAppchannelId;
+        $webhook->url       = $webhookUrl;
+
+        try {
+            $response = $this->messageBirdClient->conversationWebhooks->create($webhook);info(['akk' => $response]);
```

Comment:
```suggestion
            $response = $this->messageBirdClient->conversationWebhooks->create($webhook);
            info(['akk' => $response]);
```

Reply:
let ma update..haha...that `akk` in logging..I just used it to debug..I will remove it..

Reply:
updated at commit 10b8752e795338dfcf26d895e067a817391dbb03..please have a look